deployment/network-operator/values.yaml

```diff
@@ -128,6 +128,23 @@ ofedDriver:
   readinessProbe:
     initialDelaySeconds: 10
     periodSeconds: 30
+  upgradePolicy:
+    # global switch for automatic upgrade feature
+    # if set to false all other options are ignored
+    autoUpgrade: true
```

Comment:
lets have the feature disabled by default.

once we get some milage on it we can enable

Reply:
Makes sense, done

Reply:
Oops, thanks for noticing - now it's changed

Reply:
Now it is fine. Thanks